.github/workflows/e2e-tests.yml

```diff
@@ -9,13 +9,22 @@ jobs:
   e2e-tests:
     runs-on: ubuntu-latest
 
+
     steps:
       - name: Checkout code
         uses: actions/checkout@v4
 
```

Comment:
```suggestion
      - uses: technote-space/get-diff-action@v6.1.2
        with:
          PATTERNS: |
            **/**.go
          FILES: |
            go.mod
            go.sum
      - uses: actions/setup-go@v5
        if: env.GIT_DIFF
        with:
          go-version: "1.23"
          cache: true
```

Reply:
Note: you need to add `if: env.GIT_DIFF` to every subsequent step and to skip it if it's not needed.